src/sentry/api/endpoints/assistant.py

```diff
@@ -81,16 +81,19 @@ def put(self, request: Request):
         useful = data.get("useful")
 
         fields = {}
-        if useful is not None:
-            fields["useful"] = useful
-        if status == "viewed":
-            fields["viewed_ts"] = timezone.now()
-        elif status == "dismissed":
-            fields["dismissed_ts"] = timezone.now()
-
-        try:
-            AssistantActivity.objects.create(user=request.user, guide_id=guide_id, **fields)
-        except IntegrityError:
-            pass
+        if status == "restart":
```

Comment:
@roggenkemper can you add a test for this?

Reply:
Should we have an enum for the status modes?

Reply:
sounds like a good idea!